src/App.js

```diff
@@ -15,20 +15,21 @@ if (Platform.OS === "android") {
 }
 
 const App = () => (
-  <View style={styles.container}>
+  <SafeAreaView forceInset={{ top: "always" }} style={styles.container}>
     <StatusBar
       animated
       barStyle="light-content"
-      backgroundColor={transparent}
+      backgroundColor={lightNavyBlueColor}
```

Comment:
I wonder if this is needed. The `backgroudColor` is only used by Android. And Android should make use of the SafeAreaView because of the `SafeAreaView.setStatusBarHeight` above. And this should mean it uses the background color of the SaveAreaView. 🤔